main.go

```diff
@@ -64,13 +80,29 @@ func main() {
 }
 
 func loadInConfig() {
+	config = Config{}
```

Comment:
Might be better to unmarshal the config to a temporary variable and set the global variable at the end?

Reply:
Agreed. Done.